<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerDialog</class>
 <widget class="QDialog" name="ServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="mp_main_layout">
     <item row="2" column="0">
      <widget class="QLabel" name="mp_label_host">
       <property name="text">
        <string>Host name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mp_edit_host">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="mp_spinbox_port">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>5880</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mp_label_name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mp_edit_name">
       <property name="text">
        <string>New Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mp_label_port">
       <property name="text">
        <string>Port number</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QHBoxLayout" name="mp_buttons_layout">
       <item>
        <spacer name="mp_spacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="mp_button_box">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <spacer name="mp_spacer2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_edit_name</tabstop>
  <tabstop>mp_spinbox_port</tabstop>
  <tabstop>mp_edit_host</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mp_button_box</sender>
   <signal>rejected()</signal>
   <receiver>ServerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_button_box</sender>
   <signal>accepted()</signal>
   <receiver>ServerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
